examples.ui.config/src/main/content/jcr_root/apps/aapm-examples/osgiconfig/config/org.apache.sling.commons.log.LogManager.factory.config~aapm.cfg.json

```diff
@@ -2,7 +2,7 @@
   "org.apache.sling.commons.log.names": [
     "com.valtech.aapm"
   ],
-  "org.apache.sling.commons.log.level": "INFO",
+  "org.apache.sling.commons.log.level": "DEBUG",
```

Comment:
No debug logging by default, I think we can remove the whole config. There is no need for an extra log file for most users.